### PR TITLE
Add com.obsproject.Studio.Plugin.OBSLivesplitOne

### PR DESCRIPTION
**Pull request by junglerobba**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [ ] ~I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*~ just a plugin, no permissions
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:** https://github.com/LiveSplit/obs-livesplit-one/issues/18#issuecomment-1364756784 (didn't get a response or any further activity in that thread, but I thought I'd try this anyway, and hand this over to the developer on request)
Edit: Also contributed upstream in the time since opening this PR 😛
- [ ] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [ ] ~Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*~ no patches

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by hfiguiere** [comment]
cc @GeorgesStavracas

File: com.obsproject.Studio.Plugin.OBSLivesplitOne.metainfo.xml
Comment (hfiguiere) [inline com.obsproject.Studio.Plugin.OBSLivesplitOne.metainfo.xml:10]:
```suggestion
  <project_license>Apache-2.0 AND MIT</project_license>
```

File: com.obsproject.Studio.Plugin.OBSLivesplitOne.yml
Comment (hfiguiere) [inline com.obsproject.Studio.Plugin.OBSLivesplitOne.yml:5]: I don't think that the right runtime

Comment (junglerobba) [reply com.obsproject.Studio.Plugin.OBSLivesplitOne.yml:5]:
```
> flatpak run com.obsproject.Studio
...
info: Distribution: "KDE Flatpak runtime" "5.15-21.08"
...
```
From what I can tell OBS seems to still be using that runtime

Comment (hfiguiere) [reply com.obsproject.Studio.Plugin.OBSLivesplitOne.yml:5]: nope

```
$ flatpak remote-info flathub com.obsproject.Studio
        ID: com.obsproject.Studio
       Ref: app/com.obsproject.Studio/x86_64/stable
      Arch: x86_64
    Branch: stable
Collection: org.flathub.Stable
  Download: 191.9 MB
 Installed: 501.7 MB
   Runtime: org.kde.Platform/x86_64/6.4
       Sdk: org.kde.Sdk/x86_64/6.4

    Commit: 817625e6baf9ff8e86467f3ebdcb2b65c47b8046bd5df9ea74afe794b60f4e48
    Parent: f503a59f4910533f9bcdfb88884782d6460e029c422a73d08cf12d3ee4d14144
   Subject: Export com.obsproject.Studio
      Date: 2023-02-04 12:32:29 +0000

```

Comment (junglerobba) [reply com.obsproject.Studio.Plugin.OBSLivesplitOne.yml]: Huh, right
Guess this is just outdated then
```
[📦 org.kde.Sdk ~]$ cat /etc/os-release 
NAME="KDE Flatpak runtime"
VERSION="5.15-21.08"
VERSION_ID="5.15-21.08"
ID=org.kde.Platform
PRETTY_NAME="KDE Flatpak runtime"
BUG_REPORT_URL=https://bugs.kde.org/

```

**Comment by hfiguiere** [comment]
bot, build com.obsproject.Studio.Plugin.OBSLivesplitOne

**Comment by hfiguiere** [comment]
bot, build com.obsproject.Studio.Plugin.OBSLivesplitOne

**Comment by hfiguiere** [comment]
@GeorgesStavracas is this good to merge?

**Comment by hfiguiere** [comment]
I'll wait on @GeorgesStavracas 

bot, build com.obsproject.Studio.Plugin.OBSLivesplitOne

**Review by GeorgesStavracas — CHANGES_REQUESTED** [review]

File: com.obsproject.Studio.Plugin.OBSLivesplitOne.metainfo.xml
Comment (CryZe) [reply com.obsproject.Studio.Plugin.OBSLivesplitOne.metainfo.xml]: I'm not sure how flatpack works. Would this conflict with another package named "LiveSplit One"?

Comment (hfiguiere) [reply com.obsproject.Studio.Plugin.OBSLivesplitOne.metainfo.xml]: only the ID needs to be unique.

**Comment by hfiguiere** [comment]
bot, build com.obsproject.Studio.Plugin.OBSLivesplitOne

**Comment by junglerobba** [comment]
I think I addressed everything, is there anything left blocking this from being merged?

**Review by GeorgesStavracas — APPROVED** [review]

**Comment by GeorgesStavracas** [comment]
Nothing stood out in my eyes, I think it's ready to go

**Comment by junglerobba** [comment]
Since I don't seem to be able to add people, can @CryZe be added as a maintainer of the repo as well?

**Comment by hfiguiere** [comment]
> Since I don't seem to be able to add people,

Only admins like @barthalion can.

**Pull request closed without merging**
